modules/user/server-ts/resolvers.js

```diff
@@ -88,8 +94,17 @@ export default pubsub => ({
 
           e.throwIf();
 
-          const [createdUserId] = await User.register({ ...input });
-          await User.editUserProfile({ id: createdUserId, ...input });
+          const passwordHash = await createPasswordHash(input.password);
+
+          const trx = await createTransaction();
+          let createdUserId;
+          try {
```

Comment:
I think all the code inside this resolver should be in a transactional `try` `catch` block

Reply:
I'm not 100% sure

Reply:
@vlasenko Also, `const trx = await createTransaction();` cannot be wrapped in `try /catch` since `trx` won't be available in the `catch` block;

Reply:
Add certificate in a transaction is complete